src/Form/Type/FormatterType.php

```diff
@@ -156,37 +191,38 @@ public function buildView(FormView $view, FormInterface $form, array $options):
 
         $view->vars['format_field_options'] = $options['format_field_options'];
 
-        $defaultConfig = $this->configManager->getDefaultConfig();
-
-        if ($this->configManager->hasConfig($defaultConfig)) {
-            $ckeditorConfiguration = $this->configManager->getConfig($defaultConfig);
-        } else {
-            $ckeditorConfiguration = [];
-        }
+        $defaultConfig = $this->ckEditorConfiguration->getDefaultConfig();
+        $ckeditorConfiguration = $this->ckEditorConfiguration->getConfig($defaultConfig);
 
         if (!\array_key_exists('toolbar', $ckeditorConfiguration)) {
             $ckeditorConfiguration['toolbar'] = array_values($options['ckeditor_toolbar_icons']);
         }
 
         if ($options['ckeditor_context']) {
-            $contextConfig = $this->configManager->getConfig($options['ckeditor_context']);
+            $contextConfig = $this->ckEditorConfiguration->getConfig($options['ckeditor_context']);
```

Comment:
```suggestion
            if (null !== $this->configManager) {
                $contextConfig = $this->configManager->getConfig($options['ckeditor_context']);
            } else {
                $contextConfig = $this->ckEditorConfiguration->getConfig($options['ckeditor_context']);
            }
```
Please check usage here.